efsity-cli/README.md

```diff
@@ -265,6 +265,39 @@ $ fct translate -m merge -rf ~/Workspace/fhir-resources/<project>/<environment>/
 -l or --locale (Optional if it can be derived from the properties file). The translation locale
 ```
 
+### Update Location Lineage Tags
+To update locations lineage tags run the command:
+
+```console
+$ fct updateLocationLineage -e /path/to/env.properties -ids /path/to/location_ids.txt -bu https://base_url.org/fhir
```

Comment:
@lincmba  Do we have a sample of the `.env`

Reply:
Added